internal/util-collection/src/main/scala/sbt/internal/util/AList.scala

```diff
@@ -91,9 +91,12 @@ object AList {
     ): N[P[A]] = f(a)
   }
 
-  type ASplit[K[L[x]], B[x]] = AList[λ[L[x] => K[(L ∙ B)#l]]]
+  /** Example: calling `AList.Split[K, Task]#l` returns the type lambda `A[x] => K[A[Task[x]]`. */
+  sealed trait Split[K[L[x]], B[x]] { type l[A[x]] = K[(A ∙ B)#l] }
```

Comment:
```suggestion
/** Example: calling `AList.SplitK[K, Task]#l` returns the type lambda `F[x] => K[F[Task[x]]`. */
sealed trait SplitK[K[F0[A]], F1[A]] { type l[F2[A]] = K[(F2 ∙ F1)#l] }
```

Reply:
I think the K there stands for "higher-kinded", so T2K is "a higher-kinded tuple 2".

I think maybe the 100% correct fix would be to rename the existing `ASplit` alias to `ASplitList`, so this one could be `ASplit`, but I didn't want to repurpose an alias.  So I just pushed the name used in the local alias in FullInstance instead.

I guess you could argue maybe that the K can mean "split the KList" (the higher-kinded (heterogenous) list).  I'll push that.  I wouldn't start using new names for the type constructors though.